simcon_project/conversation_templates/models/template_node_response.py

```diff
@@ -5,6 +5,10 @@
 class TemplateNodeResponse (models.Model):
     id = models.UUIDField(unique=True, editable=False, primary_key=True, default=uuid.uuid4)
     transcription = models.CharField(max_length=1000)
-    template_node = models.ForeignKey('conversation_templates.TemplateNode',default=0, related_name='responses', on_delete=models.DO_NOTHING)
-    parent_template_response = models.ForeignKey('conversation_templates.TemplateResponse',default=0, related_name='node_responses', on_delete=models.CASCADE)
+    template_node = models.ForeignKey('conversation_templates.TemplateNode',default=0, related_name='responses',
+                                      on_delete=models.DO_NOTHING)
+    parent_template_response = models.ForeignKey('conversation_templates.TemplateResponse',default=0,
+                                                 related_name='node_responses', on_delete=models.CASCADE)
     position_in_sequence = models.IntegerField()
+    feedback = models.CharField(max_length=1000, default='No feedback')
```

Comment:
do we want a default value to be 'No Feedback' or wouldnt it be better to keep it empty and if it isn't empty display it?

Reply:
I think No Feedback works better in general, always something to display, and the student knows there's nothing instead of it not loading or something

Reply:
@Likhovodov

Reply:
I'd leave the default as empty. This leaves us more flexibility since we can still add 'No Feedback' as the default _displayed_ value later if we decide that's a good idea. I mean we can have a different displayed value with this default as well but it would be messier to check against 'No Feedback'.